octobot_backtesting/api/backtesting.py

```diff
@@ -13,24 +13,19 @@
 #
 #  You should have received a copy of the GNU Lesser General Public
 #  License along with this library.
+from octobot_backtesting.independent_backtesting import IndependentBacktesting
 from octobot_commons.constants import CONFIG_ENABLED_OPTION
 
 from octobot_backtesting.backtesting import Backtesting
 from octobot_backtesting.constants import CONFIG_BACKTESTING, CONFIG_BACKTESTING_DATA_FILES
 
 
-def create_backtesting(config, data_files) -> Backtesting:
-    return Backtesting(config, data_files)
+def create_independent_backtesting(config, data_files) -> Backtesting:
```

Comment:
A backtesting should always be independent from the current bot isn't it ?

Reply:
Right, I think it's better to explicitly have it in the method's name not to confuse it with a backtesting mode bot run